source/win-main/file-manager/file-item.vue

```diff
@@ -230,17 +233,24 @@ export default defineComponent({
       }
       return this.obj.children.filter((e: any) => [ 'file', 'code' ].includes(e.type)).length + ' ' + trans('system.files') || 0
     },
-    countWords: function () {
+    countWordsOrCharsOfDirectory: function () {
       if (this.obj.type !== 'directory') {
         return 0
```

Comment:
This is not your fault, but while we're at it, I think returning an empty string here is better … I don't know why I returned a 0!